commcare_connect/templates/tailwind/components/cards/program_card.html

```diff
@@ -0,0 +1,50 @@
+<div class="program-comp flex flex-col p-6 rounded-xl shadow-sm bg-white gap-5">
+  <div class="flex w-full justify-between">
+    <div class="flex items-center gap-2 font-normal">
+      <p class="text-xs text-gray-400 hidden">{{ date }}</p>
+      <div class="h-2 w-2 bg-brand-marigold rounded-full"></div>
+      <p class="text-xs text-gray-400">{{ organization_name }}</p>
+    </div>
+    <div class="flex items-center justify-end gap-6">
+      {% if status == 'invited' %} {% include "ccc_app/components/badges/badge_md.html" with text='invited' color='brand-deep-purple'
```

Comment:
All of this can be simplified had we used @apply directive to define badges. For e.g.

```
{% if status == 'invited' %}
    <div class="badge-md badge-invited">Invited</div>

{% elif status == 'applied' %}
    <div class="icon-button icon-blue-light">
        <i class="fa fa-download"></i>
    </div>
    <div class="badge-md badge-applied">Applied</div>

{% elif status == 'accepted' %}
    <div class="icon-button icon-deep-purple">
        <i class="fa fa-download"></i>
    </div>
    <div class="badge-md badge-accepted">Accepted</div>
{% endif %}
```